index.html

```diff
@@ -14,14 +14,40 @@
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700&display=swap" rel="stylesheet">
 </head>
 <body>
   <header>
     <div class="container">
+      <h3>Code Journal</h3>
     </div>
   </header>
   <main class="container">
-    <div data-view="entry-form"></div>
+    <div data-view="entry-form">
+      <h1>New Entry</h1>
+      <form class="row">
+        <div class="column-full column-half img-preview">
```

Comment:
You don't need to apply `column-full`, because `column-half` will already take up 100% on mobile & desktop